### ISSUE TITLE
 Is it possible to change the date format on the calendar view column heading?

### ISSUE THREAD

**Issue by asperitigroup** [YetiForceCompany/YetiForceCRM#13667]
### In the graphical calendar week view, the day column heading show the date in the format Dayname MM/DD e.g. Wed 10/21 is October 21st.  Can this be changed to display Dayname DD/MM e.g. Wed 21st October - Wed 21/10

![image](https://user-images.githubusercontent.com/58600791/96469868-9c483580-1225-11eb-9a04-3eaab09eee1c.png)

**Comment by davide-alghi** [comment]
Hi @asperitigroup,
that is not possible via YF GUI.

You have to change YF core-code, a javascript file; then **minify it and replace the original one (CalendarView.min.js)**.
**This operation is a risk: if you make a mistake, calendar will no longer work - I advise against it**
Moreover the javascript file, probably, will be overwritten by the next YF update/upgrade

anyway ...

```JS
columnFormat: 'ddd d/M',
```
after
https://github.com/YetiForceCompany/YetiForceCRM/blob/ac18f1837d6acdb7d3923a4e231b4fb14200f52c/public_html/layouts/basic/modules/Calendar/resources/Extended/CalendarView.js#L87

**Comment by asperitigroup** [comment]
Thanks for explaining this. Is there a "safe" way to try this? E.g. Save a copy of the original CalendarView.min.js file so I can make sure the modified (and minified) js file works correctly?
Is there a preferred minifier tool to use? 
I understand that the modified file may be overwritten by an upgrade, but the mod could be re-applied I assume?
Thanks in advance. 
Steve

**Comment by davide-alghi** [comment]
> Thanks for explaining this. Is there a "safe" way to try this? E.g. Save a copy of the original CalendarView.min.js file so I can make sure the modified (and minified) js file works correctly?

you can use original CalendarView.js:
- rename CalendarView.min.js in CalendarView.min.js.orig
- duplicate CalendarView.js
- rename the copy to CalendarView.min.js
- make changes to CalendarView.min.js

> Is there a preferred minifier tool to use?

https://github.com/YetiForceCompany/YetiForceScripts/tree/master/YetiDevsTools

> I understand that the modified file may be overwritten by an upgrade, but the mod could be re-applied I assume?

yes


> Thanks in advance.
> Steve

**Comment by asperitigroup** [comment]
Many thanks for the further information, I'm a bit nervous to try this as I'm not a programmer by any means, and I don't want to mess up my YF installation!  I think I might live with it for now and submit a feature request for version 6.

Best regards
Steve

**Comment by TonyM15** [comment]
Never touch code without 2 backups.

**Comment by asperitigroup** [comment]
I decided that discretion was the better part of valour. I'm not going to try to do a manual code edit to resolve this, but I have put in a feature request, so hopefully it will find its way into a Version 6 or higher update. I did look in to how to edit the Javascript files but it looked far too easy to make a mistake that would break my YF installation, and as it is my live business critical instance, I decided that this would be foolhardy.
